sort/js/search.js

```diff
@@ -0,0 +1,44 @@
+function getArray(size) {
+  var RANDOM_STEP = 8;
+  var array = [size];
+  array[0] = Math.floor(Math.random() * (-size * RANDOM_STEP >> 2));
+  for (var i = 1; i < size; i++) {
+    array[i] = array[i - 1] + Math.floor(Math.random() * RANDOM_STEP + 1);
+  }
+  return array;
+}
+
+function isSorted(items) {
+  if (!Array.isArray(items)) return;
+
+  var prev = -Infinity;
+  for (var i = 0; i < items.length; i++) {
+    if (prev > items[i]) return false;
+    prev = items[i];
+  }
+  return true;
+}
+
+function isInvalid(items, value) {
```

Comment:
let's make a module for search where `isInvalid` will be private method and won't be available in global scope
```
var binarySearch = (function () {
  recursive
  cycle
  _isInvalid

  return {
    recursive
    cycle
  }
})()
```
short variant